misc/find_prime_numbers.c

```diff
@@ -0,0 +1,134 @@
+/*
+This program will find prime numbers upto 2,000,000,000 (2 Billion)
+Here we have option to either
+1. print all prime numbers in console,
+2. show total number of prime exist till given number
+3. or create a txt file and write prime numbers in that txt file
+*/
+
+#include <stdio.h>
+#include <stdlib.h>
+
+#define MAX_LIMIT 2000000000
+#define FILE_NAME "prime_numbers.txt"
+
+void find_prime_number(unsigned long long int max);
+void write_file(int *prime, unsigned long long int max);
+
+int main()
+{
+    unsigned long long int a;
+    printf("Enter max number: ");
+    scanf("%llu", &a);
+
+    find_prime_number(a);
+
+    return 0;
+}
+
+// method to find prime number
+// max is the maximum limit to find prime numbers
+void find_prime_number(unsigned long long int max)
```

Comment:
Functions should be documented explaining the use of its parameters, a brief description, and what it returns.